config/routes.yaml

```diff
@@ -43,6 +43,16 @@ add_post:
    controller: App\Controller\FeedController::createPost
    methods: POST
 
+form_comment:
+   path: /add-comment
+   controller: App\Controller\FeedController::createComment
+   methods: GET
+
+add_comment:
+   path: /post/{id}/add-comment
+   controller: App\Controller\FeedController::createComment
+   methods: POST|GET
+
```

Comment:
Is this duplication intentional? You have the GET method on both of these route definitions

Reply:
Ah, I've removed the form-comment route